src/op_mode/scan_all_ports.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
```

Comment:
I can't see why this script is needed. Python startup time is notoriously slow, and this script only takes one argument that could easily be appended to the base command. It doesn't provide machine-friendly op mode either (we don't have a generic "perform" or "initiate" op mode word yet).

We need to either figure out a way to make this machine-friendly and use Python or just use a shell command for the time being (I'd prefer the former).